docs/examples/01-exploring-wellcome-collections-apis.md

```diff
@@ -1,6 +1,6 @@
 # 1. Exploring Wellcome Collection's APIs
 
-[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/main/notebooks/01-exploring-wellcome-collections-apis.ipynb)
+[View on GitHub](https://github.com/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/01-exploring-wellcome-collections-apis.ipynb) | [Run in Google Colab](https://colab.research.google.com/github/wellcomecollection/developers.wellcomecollection.org/blob/upgrade-everything/notebooks/01-exploring-wellcome-collections-apis.ipynb)
```

Comment:
I don't know why this happens? [Seems to be done automatically by wecobot](https://github.com/wellcomecollection/developers.wellcomecollection.org/pull/37/commits/ce345f0240e3f1a84550402a38f19ab7a857e9b2)? I'm guessing it's for internal nav and hopefully gets re-changed on merge?

Reply:
Robert confirmed it does. All good.